src/serac/numerics/functional/geometric_factors.hpp

```diff
@@ -20,22 +21,22 @@ struct GeometricFactors {
    * @brief calculate positions and jacobians for quadrature points belonging to
    * elements with the specified geometry, belonging to the provided mesh.
    *
-   * @param mesh the mesh
+   * @param domain the domain of integration
    * @param q a parameter controlling the number of quadrature points per element
    * @param elem_geom which kind of element geometry to select
    */
-  GeometricFactors(const mfem::Mesh* mesh, int q, mfem::Geometry::Type elem_geom);
+  GeometricFactors(const Domain& domain, int q, mfem::Geometry::Type elem_geom);
```

Comment:
Does this mean we can no longer upstream this to MFEM?

Reply:
In practice, it would have required some modification anyway to get it accepted into mfem, so a small change like this is easy to revert (or adapt to `mfem::SubMesh`?).